examples/operator_tests/operator_tests.ino

```diff
@@ -0,0 +1,91 @@
+#include <DS3231.h>
+
+void setup() {
+  // put your setup code here, to run once:
+  
+  Serial.begin(9600);
+  while(!Serial){};
+  
+  Serial.print("Testing Subtraction: 1 Jan 2000 00:00:01 - 1 Jan 2000 00:00:00 = ");
+  Serial.print(DateTime(946684801) - DateTime(946684800));Serial.println(" second");
+
+  Serial.print("Testing Subtraction: 1 Jan 2000 00:01:01 - 1 Jan 2000 00:00:00 = ");
+  Serial.print(DateTime(946684861) - DateTime(946684800));Serial.println(" seconds");
+
+  Serial.print("Testing Subtraction: 1 Jan 2000 00:00:00 - 1 Jan 2000 00:00:01 = ");
+  Serial.print(DateTime(946684800) - DateTime(946684801));Serial.println(" seconds");
+
+  DateTime result = DateTime(946684801) + 60;
+  Serial.print("Testing Addition: 1 Jan 2000 00:00:01 + 60 seconds = ");
+  Serial.print(result.day());Serial.print(' ');
+  Serial.print(result.month());Serial.print(' ');
+  Serial.print(result.year());Serial.print(' ');
+  Serial.print(result.hour());Serial.print(':');
+  Serial.print(result.minute());Serial.print(':');
+  Serial.println(result.second());
+
+  Serial.println("TESTING >");
+
```

Comment:
In your Tests you mainly check agains `>` or `<`, why don't check the exact values?